### ISSUE TITLE
Accessibility: `btn-link` does not have focus outline

### ISSUE THREAD

**Issue by jimmieego** [Esri/calcite-web#1052]
I noticed the example `btn-link` in the buttons section (http://esri.github.io/calcite-web/documentation/components/#buttons) does not have focus outline when in focus. Probably it's because the `outline` is set to `none`? 

This affects the New folder button in ArcGIS Online.

![screen shot 2019-01-23 at 2 18 30 pm](https://user-images.githubusercontent.com/5437651/51640956-c8803680-1f19-11e9-8f7d-632a8708fab5.png)

**Comment by paulcpederson** [comment]
also applies to `radio-group` component
